keyboards/planck/keymaps/experimental/keymap.c

```diff
@@ -279,21 +280,12 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
         break;
       return false;
         case PLOVER:
-          if (record->event.pressed) {
+          if (!record->event.pressed) {
```

Comment:
That is correct. Ideally `PREVENT_STUCK_MODIFIERS` is set and this problem is avoided. I think I may have mitigated this problem so I can probably change it back.